.github/workflows/ci.yml

```diff
@@ -18,29 +18,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        name: [ubuntu-latest, macOS-arm-latest]
-        rust: [nightly, stable]
+        name: [ubuntu-latest]
+        rust: [nightly]
```

Comment:
fwiw, `rust: [stable]` + `name: [ubuntu-latest]` seems to be the matrix that is failing on other branches.

Reply:
ugh, never mind. Ran this with stable & it's the same: no error....